Source/System.swift

```diff
@@ -45,15 +45,24 @@ internal protocol SystemType: Configurable {
 
 internal final class System: SystemType {
     
+    private var adIdManager: ASIdentifierManagerType
```

Comment:
Are there any issues with mocking a `System` for tests now that `SystemType` does not declare `adIdManager`?

Reply:
No issues, I just needed to include the `adIdManager` property to `TestSystem`

Reply:
Note that the adIdManager is still passed into the System constructor—so the output on the system instance `advertisingId` is still constrolled by the injected adIdManager.